tests/integration-tests/configs/common/common.yaml

```diff
@@ -465,6 +465,12 @@ schedulers:
         instances: {{ common.INSTANCES_DEFAULT_X86 }}
         oss: ["alinux2"]
         schedulers: ["slurm"]
+  test_slurm.py::test_scontrol_reboot:
+    dimensions:
+      - regions: ["us-east-1"]
```

Comment:
Is it necessary to execute this test in us-east-1? If possible it would be better to distribute tests also in other regions.

Reply:
Discussed separately: we are going to set `us-west-2`. Addressing in the next commit.